### PR TITLE
Create a class function to AmazonEmailService and organized params as record

### LINKED ISSUE #12: Create a class function that instantiate the AmazonEmailService and send the email automatically

### PR DESCRIPTION
**Pull request by martinusso**
Fixes #12

### CODE REVIEW

**Comment by fabioxgn** [comment]
Way better :)

:+1:  :shipit:

File: AmazonEmailService.pas [hunk 1]
```diff
@@ -184,4 +154,29 @@ function TAmazonEmailService.SendMail(const Recipients, ReplyTo: TArray<string>;
...
 end;
 
+class function TAmazonEmailService.SendMail(const EmailMessage: TEmailMessage): Boolean;
+var
+  Response: TCloudResponseInfo;
+begin
+  try
+    Result := TAmazonEmailService.SendMail(EmailMessage, Response);
+  finally
+    if Assigned(Response) then
```
Comment (esasse) [inline AmazonEmailService.pas]: Wrong.

File: tests/BuildQueryParametersTests.pas [hunk 2]
```diff
@@ -35,8 +33,8 @@ implementation
 
 procedure TBuildQueryParametersTests.AddRecipient(const Address: string);
 begin
-  SetLength(FRecipients, Length(FRecipients) +1);
-  FRecipients[High(FRecipients)] := Address;
+  SetLength(FEmailMessage.Recipients, Length(FEmailMessage.Recipients) +1);
```
Comment (esasse) [inline tests/BuildQueryParametersTests.pas]: Space after `+`.

**Comment by esasse** [comment]
Please update readme.

**Comment by martinusso** [comment]
done @esasse

**Comment by esasse** [comment]
Thanks! :+1:

**Pull request merged**